app/bundles/EmailBundle/Helper/FormSubmitHelper.php

```diff
@@ -54,12 +54,16 @@ public static function sendEmail($tokens, Action $action, MauticFactory $factory
 
 				$currentLead = array_merge($currentLead, $leadFields);
 			}
+            /** @var \Mautic\EmailBundle\Helper\MailHelper $emailHelper */
+            $emailHelper = $factory->getHelper('email.email');
 
-			if (isset($properties['user_id']) && $properties['user_id']) {
+            $frequencyRules = $emailHelper->applyFrequencyRules($lead);
+
+			if (isset($properties['user_id']) && $properties['user_id'] && $frequencyRules === true) {
```

Comment:
This is for the send to user form submit and so a lead/frequency rules won't be applicable here